src/NextGen/dataHandler/dataHandler.ts

```diff
@@ -0,0 +1,7 @@
+import { IMParticleComponent } from '../core/component';
+
+export interface IMParticleDataHandler extends IMParticleComponent {
+    // QUESTION: Shouldn't we be passing in a type for data instead of being "any"?
```

Comment:
Maybe Mark was using 'any' to represent it necessarily being a different type of data depending on platform? @markvdouw ?

Reply:
If we want to make this component 100% agnostic for the community, we need to be able to pass ANY and then define within each strategy how to map that ANY to a specific model.
If we want to scope it within MParticle, we can define a little bit better using Generics